algosdk/atomic_transaction_composer.py

```diff
@@ -479,6 +479,55 @@ def submit(self, client: algod.AlgodClient) -> List[str]:
         self.status = AtomicTransactionComposerStatus.SUBMITTED
         return self.tx_ids
 
+    def simulate(
+        self, client: algod.AlgodClient
+    ) -> "SimulateAtomicTransactionResponse":
```

Comment:
nit: If you define `class SimulateAtomicTransactionResponse` above `AtomicTransactionComposer`, you should be able to refer to it without these quotes.

Reply:
I tried to make it match the other methods here, should I do this and also move the stuff returned from `execute` as well?

Reply:
@tzaffi thumbs up mean do it?

It'll need to be all of the classes below AtomicTransactionComposer though so will be a large diff

Reply:
I'm not bothered by a large diff here, but most people probly are. We can kick this can as a follow up that simply re-arranges the order as it's own PR.